build

```diff
@@ -12,29 +12,23 @@ if [ -z ${VERSION+a} ]; then
 	VERSION=$(git describe --dirty --always)
 fi
 
-GLDFLAGS+="-X github.com/flatcar-linux/ignition/internal/version.Raw=${VERSION}"
-
-if [ ! -h gopath/src/${REPO_PATH} ]; then
-	mkdir -p gopath/src/${ORG_PATH}
-	ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
-fi
```

Comment:
Seems like we could get rid of REPO_PATH in separate commit, so we could make the module renames atomic, so each commit compiles.